runtime/JavaScript/src/antlr4/CommonToken.js

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
+ * Use of this file is governed by the BSD 3-clause license that
+ * can be found in the LICENSE.txt file in the project root.
+ */
+
 import Token from "./Token.js";
```

Comment:
Please remove the extension

Reply:
Two points here:

1) The .js extension is used already in the JS runtime in many places.
2) The .js extension is required for [full ESM support](https://nodejs.org/api/esm.html#mandatory-file-extensions).

Reply:
Yes sorry, this is a js file